includes/Context.php

```diff
@@ -324,6 +324,12 @@ public function is_amp() {
 	 */
 	public function get_amp_mode() {
 		if ( ! class_exists( 'AMP_Theme_Support' ) ) {
+			// If the AMP plugin isn't enabled but this page/post is a web
+			// story, use AMP Secondary mode.
+			if ( is_singular( 'web-story' ) ) {
```

Comment:
@tofumatt In addition to @aaemnnosttv's comment, per https://github.com/google/site-kit-wp/issues/2998#issuecomment-869865609 this shouldn't check whether the current context is a web story (which will _never_ be the case in WP Admin), instead it needs to check whether the Web Stories plugin is active (e.g. check for whether the `WEBSTORIES_VERSION` constant is defined (see https://github.com/google/web-stories-wp/blob/main/web-stories.php).